### PR TITLE
Add Support For Parquet Read in SQS Ingestion

### PR DESCRIPTION
**Pull request by atris**

### CODE REVIEW

**Review by kelindar — CHANGES_REQUESTED** [review]

File: internal/server/server.go [hunk 1]
```diff
@@ -138,9 +139,15 @@ func (s *Server) pollFromSQS(conf *config.Config) (err error) {
 	// Start ingesting
 	s.monitor.Info("server: starting ingestion from S3/SQS...")
 	s.s3sqs.Range(func(v []byte) bool {
```
[the remark this answers is not in the document]
Comment (atris) [reply internal/server/server.go:140]: @kelindar I do not see a way to get the file extension type from the SQS message, could you please help with that?

[the remark this answers is not in the document]
Comment (qiaowei-g) [reply internal/server/server.go:140]: How about modifying the handler function signature to `func(url string) bool` and use 
`Data: &talaria.IngestRequest_Url{Url: url}}` in the ingestRequest to handle different file extensions as well as loading data? The handler is called in https://github.com/kelindar/talaria/blob/master/internal/ingress/s3sqs/s3sqs.go#L156, there are the bucket and key.

[the remark this answers is not in the document]
Comment (atris) [reply internal/server/server.go:140]: Challenge is -- how do I get the file extension?

[the remark this answers is not in the document]
Comment (qiaowei-g) [reply internal/server/server.go:140]: I assumed [the s3 key](https://github.com/kelindar/talaria/blob/master/internal/ingress/s3sqs/s3sqs.go#L146) should contain the file extension

[the remark this answers is not in the document]
Comment (ocassetti) [reply internal/server/server.go:140]: If I remember well this is the structure of the message:
https://docs.aws.amazon.com/AmazonS3/latest/userguide/notification-content-structure.html

`            "object":{  
               "key":"object-key",
`
If the producer writes the object with an extension e.g. `.orc` `.parquet`  then you should be fine you can do something along the lines of 
```
package main

import (
	"fmt"
	fs "path/filepath"
)

func main() {
	fmt.Println(fs.Ext("test.orc"))
}
```

[the remark this answers is not in the document]
Comment (atris) [reply internal/server/server.go:140]: Thanks folks, I opened a new PR with the said approach:

https://github.com/kelindar/talaria/pull/74

Please review

**Comment by atris** [comment]
Superseded by https://github.com/kelindar/talaria/pull/74

**Pull request closed without merging**